website/app/components/doc/image-caption/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class DocImageCaptionComponent extends Component {
+  constructor() {
+    super(...arguments);
+  }
+
+  get classNames() {
+    const classes = ['doc-image-caption'];
```

Comment:
When there is only one class name, or there is no "logic" associated with the class names, we write directly the classes in the Handlebars template. See for example here: https://github.com/hashicorp/design-system/blob/main/website/app/components/doc/npm-version.hbs#L6

Reply:
Understood will follow this pattern going forward!